yaml/index.adoc

```diff
@@ -82,12 +82,16 @@ include::../examples/my-site.yaml[]
 ----
 The YAML creates a site with a console and you can create tokens from this site.
 
-To create a site that has no ingress:
+To create a site that has no ingress and with `service-sync` disabled:
 
 ----
 include::../examples/no-ingress.yaml[]
 ----
 
+NOTE: By default, `service-sync` is enabled and if you create a service on one site, it is available on all sites.
```

Comment:
I'm not sure about using 'available' here.  The skupper service is still available on all sites.  The difference is that without the service-sync a k8s service will not be automatically created on this site.

Perhaps something along the lines

> By default `service-sync` is enabled, and a Kubernetes service is automatically created locally for each/every/all skupper services present on the VAN